src/main/java/io/github/microcks/util/openapi/OpenAPIImporter.java

```diff
@@ -71,7 +71,7 @@ public OpenAPIImporter(String specificationFilePath) throws IOException {
                isYaml = false;
                break;
             }
-            else if (line.startsWith("---") || line.startsWith("openapi: 3")) {
+            else if (line.startsWith("---")) {
```

Comment:
Revert #126 that was legit because starting with dashes is not mandatory in YAML

Reply:
Something like `|| line.startsWith("openapi: ")` would not invalidate #126 and make the check less strict. Would you want to update the PR ? Or I'll push something on my side ?

Reply:
Oooh okay, yeah that makes sense. I can update this today, sorry I didn't notice the failing test